pkg/virt-launcher/virtwrap/nichotplug.go

```diff
@@ -95,6 +97,40 @@ func (vim *virtIOInterfaceManager) hotplugVirtioInterface(vmi *v1.VirtualMachine
 	return nil
 }
 
+func (vim *virtIOInterfaceManager) syncInterfacesLinkState(currentDomain, desiredDomain *api.Domain) error {
+
+	currentDomainIfacesByAlias := indexedDomainInterfaces(currentDomain)
+	for _, desiredIface := range desiredDomain.Spec.Devices.Interfaces {
+		curIface, ok := currentDomainIfacesByAlias[desiredIface.Alias.GetName()]
+		if !ok {
+			return fmt.Errorf("could not find requested domain interface %s in current domain", desiredIface.Alias.GetName())
+		}
+
+		if !isLinkStateEqual(curIface, desiredIface) {
+			curIface.LinkState = desiredIface.LinkState
+			if err := vim.updateDomainWithLinkState(&curIface); err != nil {
+				return err
+			}
+		}
+
+	}
+	return nil
+}
+
+func (vim *virtIOInterfaceManager) updateDomainWithLinkState(domIfaceToUpdate *api.Interface) error {
```

Comment:
This function's functionality was reduced to just updating the interface in the domain.
Please consider adjusting it name to better reflect its purpose.

Reply:
Not done (at least in the commit I see this comment on).

Reply:
Consider renaming it to `updateIfaceInDomain` or something along those lines.